cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1806,6 +1806,195 @@ search = | savedsearch "Palo Alto Firewall - Network Compromise - DDoS Attack Pr
 action.cyences_notable_event_action.products = Palo Alto
 
 
+# =============================
+# Fortinet Fortigate Firewall
+# =============================
+
+[Fortigate Firewall - Network Compromise - Fortigate DNS Sinkhole]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 5
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A Fortigate Firewall DNS alert for when outbound traffic goes to 72.5.65.111. \
+\
+Data Collection - Fortigate Networks firewalls traffic data (Fortinet Fortigate Add-on for Splunk).
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_fortigate` (sourcetype="fortigate_traffic" OR sourcetype="fortigate_utm") dest_ip="72.5.65.111" \
```

Comment:
Are we sure about this IP for fortigate for DNS sinkhole?

Reply:
No, this needs to decide. I kept just for testing.